assets/blocks/course-outline/module-block/module-block.scss

```diff
@@ -64,17 +64,6 @@ $block: '.wp-block-sensei-lms-course-outline-module';
 		text-transform: uppercase;
 	}
 
-	&__name__minimal-border {
-		display: block;
-		border: none;
-		height: 4px;
-		.wp-block-sensei-lms-course-outline.is-style-default & {
```

Comment:
I wonder if we should keep this margin? In the editor a margin is being applied, but on the front end it's not.

Reply:
Hmm not sure if there is a case that the margin is applied. The minimal-border exists in the frontend only and only when the class is not `is-style-default`. This was also discussed (and removed) [here](https://github.com/Automattic/sensei/pull/3741#discussion_r528981941)